hudi-hadoop-mr/src/test/java/org/apache/hudi/hadoop/testutils/InputFormatTestUtil.java

```diff
@@ -175,8 +182,12 @@ public static File prepareParquetTable(java.nio.file.Path basePath, Schema schem
   public static File prepareParquetTable(java.nio.file.Path basePath, Schema schema, int numberOfFiles,
                                          int numberOfRecords, String commitNumber, HoodieTableType tableType) throws IOException {
     HoodieTestUtils.init(HoodieTestUtils.getDefaultHadoopConf(), basePath.toString(), tableType, HoodieFileFormat.PARQUET);
+
```

Comment:
nit: remove the added empty lines in several places?

Reply:
Reason why i'm adding those is to logically separate clusters of operations (like setting up partition, inserting data, etc) to make it easier to digest from just a glance. WDYT?

Reply:
Got it, Sg.  I'll let you make the judgement.